pages.zh/common/amass.md

```diff
@@ -0,0 +1,21 @@
+# amass
+
+> 深度攻击面探测与资产发现工具。
+> 部分子命令如 `amass db` 拥有独立的文档。
```

Comment:
We have a [template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md) for it.
```suggestion
> 此命令也有关于其子命令的文件，例如：`amass db`.
```